src/kernel/capinv.c

```diff
@@ -547,10 +536,13 @@ asnd_process(struct thread *rcv_thd, struct thread *thd, struct tcap *rcv_tcap,
 	thd_rcvcap_pending_inc(rcv_thd);
 	next = notify_process(rcv_thd, thd, rcv_tcap, tcap, tcap_next, yield);
 
-	if (next == thd)
-		tcap_wakeup(rcv_tcap, tcap_sched_info(rcv_tcap)->prio, 0, rcv_thd, cos_info);
-	else
-		thd_next_thdinfo_update(cos_info, thd, tcap, tcap_sched_info(tcap)->prio, 0);
+	/*
+	 * FIXME: Need to revisit the preemption-stack functionality
+	 */
+	//if (next == thd)
```

Comment:
If you aren't going to remove this, at least use proper comments (`/* ... */`).

Reply:
kk, i'll.